Conclusion-rewrite (#45)
greenelab/annorxiver_manuscript: content/05.conclusion.md

## Discussion and Conclusions

> BioRxiv is a popular repository that contains life science preprints; however, the language contained within these preprints remains unknown.

Reviewer: ```suggestion
BioRxiv is a repository that contains life science preprints; however, the language contained within these preprints has not been systematically examined.
```

> Throughout this work we sought to analyze the language contained within these preprints and understand how it changes in response to the peer review process.

Reviewer: ```suggestion
Throughout this work we sought to analyze the language contained within these preprints and understand how it changes in response to peer review.
```

Through our global corpora analysis, we found that writing within bioRxiv is consistent with the biomedical literature contained in the PMCOA repository, suggesting that bioRxiv is linguistically similar to PMCOA.
Token-level analyses between bioRxiv and PMCOA suggested that major differences are driven by research fields where more patient related research is prevalent in PMCOA than bioRxiv.
This observation is expected as patient preprints are funneled from bioRxiv into the medRxiv repository [@url:https://www.medrxiv.org/].

> Through our document embedding approach., we observed that life science preprints were separated on an informatics / molecular axis, suggesting that majority of life science research involves some form of a cellular biology or an informatics type of approach.

Reviewer: ```suggestion
Examining document embeddings revealed that the largest source of variability for life science preprints was an informatics / molecular axis.
```
I'm not sure the second part of that statement is really true because you show above that bioRxiv and PMC don't really have quite the same content.

We used the full text of documents to correct these missing annotations and our analysis provided a comprehensive understanding of the extent to which preprints are published.
This correction resulted in a publication rate that is higher than previously estimated.
Importantly, our analysis only accounts for papers that are published open access, so these results should be considered to raise the lower bound of the total preprint publication fraction; however, the true fraction is likely to be much higher.
In any case, future work, especially that which aims to assess the fraction of preprints that are eventually published, should account for the possibility of missed annotations.

Our work presents an early step towards understanding the process by which peer review alters life sciences papers.
We analyzed and compared of the full text of preprints with their corresponding publication.
Our analysis revealed differences in typesetting and supplementary materials, suggesting that the majority of peer review changes are modest at best. 
Our results lay out the groundwork for future endeavors that aims to identify sentences and claims that are altered during the peer review process.

> Preprints in general take approximately 16 days on average to venture through the peer review process provided that modest changes are requested.

Reviewer: I don't think your results show this. What's the average time on peer review? Why is a document distance of 1 the average?

> We also discovered that the peer review process is agnostic to preprint categories overall with the exception being the scientific communication and education category.

Reviewer: Also not sure that your results show this. What do you mean by agnostic?

Future investigation is required to uncover the reasons behind this large delay.

Lastly, we found that document embeddings were associated with the eventual journal at which the work was published.
We trained two machine learning models to identify which journals publish similar papers towards a query preprint.

> Our models achieved modest results, which proposes that future models should include more information than solely textual content when identifying which journal venues are appropriate for a preprint.

Reviewer: Why do you argue the results are modest? In terms of fold change over random, they are quite good. I'm not sure what you could expect as a top-line accuracy given that the noise in the system has to be pretty high as it is.
